docs/index.md

```diff
@@ -4360,6 +4360,24 @@ You can follow
 [commits in its `develop` branch](https://github.com/expln/metamath-lamp/commits/develop) and even try out the
 [development version of metamath-lamp application page (but this may not work as expected)](https://expln.github.io/lamp/dev/index.html).
 
+Features currently implemented in the development version include:
+
+* [Goal labels are validated as strictly as hypothesis labels](https://github.com/expln/metamath-lamp/issues/81)
+* [Adding a "reset" option](https://github.com/expln/metamath-lamp/issues/21)
+* [Consistently determining replacement fields](https://github.com/expln/metamath-lamp/issues/82#issuecomment-1666603023)
+* [Make it more obvious how to get a completed proof](https://github.com/expln/metamath-lamp/issues/11#issuecomment-1666023918)
+* [Enable easily preloading an existing proof](https://github.com/expln/metamath-lamp/issues/8). Note that this automatically merges duplicate statements (just like metamath.org view does).
```

Comment:
`Note that this automatically merges duplicate statements (just like metamath.org view does).` - I think no need to mention this because I changed this in both preloaded proof and in the proof table on the individual theorem tabs. So users will not see this automatic merge. Technically, it is even not a merge. I updated the algorithm to use index of previously defined row with the same statement and same justification. If there are two rows with same statement but different justifications then both rows will be shown. But metamath.org does the same (at least in those few cases I checked on metamath.org).

Reply:
Sure!